apps/wfo-ui/components/Metadata/Products/Products.tsx

```diff
@@ -99,20 +106,33 @@ export const Products: FC<ProductsProps> = ({
                 </>
             ),
         },
+        fixedInputs: {
+            field: PRODUCT_FIELD_FIXED_INPUTS,
+            name: t('fixedInputs'),
+            render: (fixedInputs) => (
```

Comment:
minor could: this whole block could be a FixedInputs component to end up wit a more compact tableColumns object here